tests/cases/hash_state_test.py

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+###############################################################################
+#  Copyright 2016 Kitware Inc.
+#
+#  Licensed under the Apache License, Version 2.0 ( the "License" );
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+###############################################################################
+
+from .. import base
+from girder.utility import hash_state
+import hashlib
+import this
+
+def setUpModule():
+    base.startServer()
+
+
+def tearDownModule():
+    base.stopServer()
+
+
+# Define a few variables used by all the tests:
+# Use the python's zen as 'long' string
+test_chunk = this.s.encode('utf8')
```

Comment:
These should be moved to a ``def setUp(self):`` method inside the test class, which assigns any relevant data onto ``self``, where it can then be used by test methods.

The reason for doing this is to ensure the code runs at the time of the test, as opposed to whenever this file happens to be imported. It also ensures that if the data is ever modified by some test method, it gets freshly re-generated for the other test methods.